src/PHPDocker/Template/docker-compose-fragments/mysql.yml.twig

```diff
@@ -10,4 +10,6 @@
         - MYSQL_DATABASE={{ mysql.getDatabaseName() }}
         - MYSQL_USER={{ mysql.getUsername() }}
         - MYSQL_PASSWORD={{ mysql.getPassword() }}
+      ports:
+        - 3306:3306
```

Comment:
Could you please change this to follow the same pattern Mailhog uses to set its open port? 

See https://github.com/phpdocker-io/phpdocker.io/blob/0b318fad38ccadbf1ae5249619edfc7e8d07a217/src/PHPDocker/Template/docker-compose-fragments/mailhog.yml.twig#L6

The reason for this is to avoid clogging default ports on the host computer. Users can always change their docker-compose files to match their requirements.